### PR TITLE
Update email subject to Valora

### PR DESCRIPTION
**Pull request by isabellewei**
### Description

Change the support email subject to `Valora` from `Celo`

### Related issues

- Fixes #https://github.com/valora-inc/support-issues/issues/35

### CODE REVIEW

File: packages/mobile/src/account/SupportContact.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ function SupportContact({ route }: Props) {
       network: DEFAULT_TESTNET,
     }
     const userId = e164PhoneNumber ? anonymizedPhone(e164PhoneNumber) : 'unknown'
-    const emailSubject = 'Celo support for ' + (userId || 'unknownUser')
+    const emailSubject = 'Valora support for ' + (userId || 'unknownUser')
```
Comment (jeanregisser) [inline packages/mobile/src/account/SupportContact.tsx:62]: Proposing to use the existing constant we have:
```suggestion
    const emailSubject = `${APP_NAME} support for ${userId || 'unknownUser'}`
```

**Review by jeanregisser — COMMENTED** [review]
Thanks!

File: packages/mobile/src/account/SupportContact.tsx (hunk 1 above)
Comment (jeanregisser) [reply packages/mobile/src/account/SupportContact.tsx:62]: By the way, we should translate this.

Comment (isabellewei) [reply packages/mobile/src/account/SupportContact.tsx]: I think the emails are sent to our support team, which is why it's just in English, do you think it needs to be translated to user's language?

Comment (jeanregisser) [reply packages/mobile/src/account/SupportContact.tsx]: Yes please.

**Review by jeanregisser — APPROVED** [review]
Thanks!

**Comment by ValoraQA** [comment]
Hi @MuckT, We have verified this issue using Android Internal Build V1.23.0 , iOS Test flight build V1.23.0 & updated status in below Gsheet.

**Sheet Link:** 
https://docs.google.com/spreadsheets/d/15Z15zu4roT6XFD1dbi9__c3Q6aQZpIiIZka1rKVrCgw/edit#gid=1644898358

**Devices:** iPhone 13 mini (15.1.1), One Plus 7T(11.0)
CC: @isabellewei 
Thanks..!!

**Comment by ValoraQA** [comment]
Hi @MuckT, We have verified this issue using Android Internal Build V1.23.0 , iOS Test flight build V1.23.0 & updated status in below Gsheet.

**Sheet Link:**
https://docs.google.com/spreadsheets/d/15Z15zu4roT6XFD1dbi9__c3Q6aQZpIiIZka1rKVrCgw/edit#gid=1644898358

**Devices:** Samsung Galaxy M40 (10.0), iPhone SE (14.2)
CC: @isabellewei
Thanks..!!

**Comment by ValoraQA** [comment]
Hi @MuckT, We have verified this issue using Android Internal Build V1.23.0 & updated status in below Gsheet.

**Sheet Link** :
https://docs.google.com/spreadsheets/d/15Z15zu4roT6XFD1dbi9__c3Q6aQZpIiIZka1rKVrCgw/edit#gid=1644898358

**Devices** : Samsung Galaxy A8+(8.0), Redmi Note 5 Pro (9.0)
CC: @isabellewei 
Thanks..!!

**Pull request merged**